src/scenarios/social-media/index.ejs

```diff
@@ -0,0 +1,25 @@
+<%- include(commonPath + '/header.ejs') %>
+
+    <%- include(commonPath + '/internal-page/header.ejs', {containerType: 'sm'}) %>
+        <div id="fb-root"></div>
+        <script
+            async
+            defer
+            crossorigin="anonymous"
+            src="https://connect.facebook.net/pt_PT/sdk.js#xfbml=1&version=v19.0&appId=222635294526394"
```

Comment:
Use English script instead for Like and Comment button to appear with English label 
```suggestion
            src="https://connect.facebook.net/en_EN/sdk.js#xfbml=1&version=v19.0&appId=222635294526394"
```